src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -261,9 +261,15 @@ export default class ResultsViewOncoprint extends React.Component<
     } = {};
 
     @computed get selectedClinicalTrackConfig(): ClinicalTrackConfigMap {
-        const clinicalTracks = _.clone(
-            this.urlWrapper.oncoprintSelectedClinicalTracks
-        );
+        const clinicallist = this.props.store.pageUserSession.userSettings
+            ?.clinicallist;
+        if (clinicallist) {
+            const map = {} as ClinicalTrackConfigMap;
+            clinicallist.forEach(cl => (map[cl.stableId] = cl));
+            return map;
+        }
+        const clinicalTracks =
```

Comment:
why are we still looking at urlWrapper here if we're using session as source of truth

Reply:
Fixed in [240ae85](https://github.com/cBioPortal/cbioportal-frontend/compare/b0bd2c653119b4363c45d9c7f7acd584ea0d6436..240ae85eba446230bffd086012e309356e71f413)